compiler/qsc_circuit/src/builder.rs

```diff
@@ -88,35 +88,35 @@ impl Backend for Builder {
 
     fn rx(&mut self, theta: f64, q: usize) {
         let q = self.map(q);
-        self.push_gate(rotation_gate("rx", theta, [q]));
+        self.push_gate(rotation_gate("Rx", theta, [q]));
```

Comment:
Do we have a central place for gate names for all of our Rust code? Should we?

Reply:
It doesn't seem like we do, and I'm not really in favor of adding one. Normally the gate names are a simple "X" or "S", and it would be pretty tedious to have to make a reference to some centralized location in code. I can see the argument that it would make gate name references safer, as demonstrated by the need for fix in this PR, but in general I think it would cause headache and be more trouble than its worth. That's just my take on it though, and I can see that it might just come down to a matter of preference.